README.md

```diff
@@ -211,34 +219,88 @@ Link to other services:
+
+2. **Input:** A payment file where the amount of payment requests does not match the batch header.
+**Output:** An error message is logged and the payment file is moved from `inbound` to `quarantine`.
+
+3. **Input:** A payment file with an invalid batch header (e.g. wrong date format)
+**Output:** An error message is logged and the payment file is moved from `inbound` to `quarantine`.
+
+4. **Input:** A payment file where the batch header value matches the payment line values but one or more payment line does not match the corresponding invoice line total.
+**Output:** The valid payment requests are logged and error messages are logged for each invalid request, the payment file is moved from `inbound` to `archive`, each valid payment request is sent to Topic `PAYMENT_TOPIC_ADDRESS`, and an invalid event is sent to `EVENT_TOPIC_ADDRESS` for each invalid payment request.
+
+5. **Input:** A payment file where the batch header sequence number is different from the expected value.
+**Output:** If the sequence number is lower than the expected value the payment file is ignored. If the sequence number is higher than the expected value the payment file is quarantined. A message is logged stating that the payment file has been ignored or quarentined accordingly.
+
+#### Example output message
+
+```
+{
+  "sourceSystem": "SFIP",
+  "frn": 1234567890,
+  "marketingYear": 2022,
+  "paymentRequestNumber": 1,
+  "correlationId":"9e016c50-046b-4597-b79a-ebe4f0bf8505",
+  "invoiceNumber": "SFI12345678",
+  "agreementNumber": "SFI12345",
+  "contractNumber": "SFI12345",
+  "currency": "GBP",
+  "schedule": "Q4",
+  "dueDate": "09/11/2022",
+  "value": 1000.00,
+  "invoiceLines": [{
+    "schemeCode": "80001",
+    "accountCode": "SOS100",
+    "fundCode": "DRD10",
+    "description": "G00 - Gross value of claim",
+    "value": 1000.00
+  }]
+}
+```
+
+# How to stop the Batch Processor
+
+Run the command `docker-compose down`
+or `docker-compose down -v` to also clear the data in the database.
+
+# How to test the Batch Processor
 ## Test structure
 
 The tests have been structured into subfolders of `./test` as per the
 [Microservice test approach and repository structure](https://eaflood.atlassian.net/wiki/spaces/FPS/pages/1845396477/Microservice+test+approach+and+repository+structure)
 
 ### Running tests
+Tests can be run in several modes
+- [Run tests and exit](#run-tests-and-exit)
+- [Run tests with file watch](#run-tests-with-file-watch)
+- [Run tests with debug](#run-tests-with-debug)
 
 A convenience script is provided to run automated tests in a containerised
 environment. This will rebuild images before running tests via docker-compose,
 using a combination of `docker-compose.yaml` and `docker-compose.test.yaml`.
 The command given to `docker-compose run` may be customised by passing
 arguments to the test script.
 
-Examples:
+#### Run tests and exit
+`scripts/test`
```

Comment:
Use code block rather than inline to be consistent across READMEs (triple backtick (`), newline code text, newline, triple backtick (`))

```
./scripts/tests
```